DAO.sol

```diff
@@ -350,6 +349,34 @@ contract DAO is DAOInterface, Token, TokenCreation {
         if (balanceOf(msg.sender) == 0) throw;
             _
     }
+    modifier creationOver{
+          if (now < closingTime) throw;
+	    _
+    }
+    modifier Fueled{
+          if(!isFueled) throw;
+            _
+    }
+    modifier notDAO {
+        if (msg.sender == address(this)) throw;
+            _
+    }
+    modifier onlyDAO {
```

Comment:
This `notDAO` for example is used only in one place where the is also a comment that explains why the check is there. By using a modifier we gain nothing and lose a lot on readability.

So I'd rather not have this as a modifier.